pallets/connectors/src/message.rs

```diff
@@ -201,6 +201,90 @@ where
 		tranche_id: TrancheId,
 		investor: Address,
 	},
+	/// The message sent back to the domain from which a `DecreaseInvestOrder`
+	/// message was received, ensuring the correct state update on said domain
+	/// and that the `investor`'s wallet is updated accordingly.
+	///
+	/// Directionality: Centrifuge -> EVM Domain.
+	ExecutedDecreaseInvestOrder {
+		/// The pool id
+		pool_id: PoolId,
+		///The tranche id
+		tranche_id: TrancheId,
+		/// The investor's address
+		investor: Address,
+		/// The currency in which `DecreaseInvestOrder` was realised
+		currency: u128,
+		/// The amount of `currency` that was actually executed in the original
+		/// `DecreaseInvestOrder` message, i.e., the amount by which the
+		/// investment order was actually decreased by.
+		decreased_by: Balance,
+		/// The outstanding order, in `currency` units
+		outstanding_order: Balance,
+	},
```

Comment:
Based on

> InvestmentDecreased
>  - How much is the outstanding order
>  - By How much it was decreased
>  - Investor’s address

Reply:
I actually don't understand still why we need `ExecutedDecreaseInvestOrder` and `ExecutedDecreaseRedeemOrder` but not the `Increase` equivalents 🤔 @ilinzweilin @mustermeiszer can you explain ?

Reply:
My understanding was that `Decrease*` and `Collect*` calls require a message being sent back to the source chain in order to (re-)fund the investor whereas `Increase*` does not require callback handling.

Reply:
Makes sense, thanks both for explaining!